File: skills/kehwar-frappe-standard-script-report-schema/assets/report.template.py
# Copyright (c) __YEAR__, __APP_PUBLISHER__ and contributors
# For license information, please see license.txt

# import frappe


def execute(filters=None):
    columns = [
        {
            "label": "Name",
            "fieldname": "name",
            "fieldtype": "Data",
            "width": 180,
        },
    ]
    data = []
    return columns, data

File: skills/frappe-standard-script-report-schema/assets/report.template.js
// Copyright (c) __YEAR__, __APP_PUBLISHER__ and contributors
// For license information, please see license.txt

frappe.query_reports["__REPORT_NAME__"] = {
    filters: [
        {
            fieldname: "company",
            label: __("Company"),
            fieldtype: "Link",
            options: "Company",
        },
    ],
};
